src/seedu/addressbook/commands/Command.java

```diff
@@ -63,11 +63,20 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    /**
+     * Check if the command is able to mutate data in address book
+     * @return false by default
+     */
+    public boolean isMutating() {
```

Comment:
is there a better way of doing this? Think about what kind of class `Command` is